lib/Centreon/Object/Relation/Host/Template/Host.php

```diff
@@ -84,18 +84,24 @@ public function delete($fkey, $skey = null): void
             parent::delete($fkey, $skey);
```

Comment:
What is fkey & skey?

Reply:
Hello Jeremy,
This method inherits the delete() method from the parent class - Relation.php in two folders up. It should be compatible with the parent definition, so I have no choice about parameters. My suggestion is that fkey and skey are first key and second key.

Reply:
Ah, okay. Okay.
I wasn't looking at the delete class() but "f" for "first" & "s" for "second", right?

We can possibly specify it even if in the parent class it is legacy. This allows you to know quickly with a small comment.